src/ff16r_strategy.cpp

```diff
@@ -91,216 +91,9 @@ FF16r_Strategy::FF16r_Strategy() {
   collect_all_auxillary = false;
   // build the string state/aux name to index map
   refresh_indices();
-  name = "FF16r";
+  name = "FF16";
```

Comment:
Is this change correct? Should name still be FF16r? Not sure we actually need or use the name anyway. I think i added before @rafaqz outlined another approach to detecting strategy type.

Reply:
Otherwise, love seeing all the duplicated code being deleted here.

Reply:
Oops. Yeah I search and replaced them all so the diff was cleaner to delete all that code, just forgot to swap that one back.